service/controller/clusterapi/v28/machine_deployment_resource_set.go

```diff
@@ -37,19 +36,6 @@ type MachineDeploymentResourceSetConfig struct {
 func NewMachineDeploymentResourceSet(config MachineDeploymentResourceSetConfig) (*controller.ResourceSet, error) {
 	var err error
 
-	var asgStatusResource controller.Resource
```

Comment:
This doesn't work for `MachineDeployment` at the moment as it's built for `Cluster`. It's also not needed at the moment so I removed it as unnecessary.